### PR TITLE
Ritwika/quickstart

### PR DESCRIPTION
**Pull request by ritwika314**

### CODE REVIEW

File: quickstart_notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,1760 @@
...
+    }
+   },
+   "source": [
+    "\n",
+    "---------------\n",
+    "\n",
+    "## Connect to a cluster \n",
+    "\n",
+    "This notebook runs code on a cluster. If you are in the Community Edition Workspace, and your notebook is *detached*, make sure the Community Edition Cluster is running. On the sidebar, right click on **<img src=\"img/cluster_icon.png\"/></a> Clusters** and open tab. If the cluster is paused, click on the play button. Once the state changes to running, the notebook should automatically attach to the cluster. If it doesn't, click the dropdown and attach to the Community Edition Cluster. \n"
+   ]
```
Comment (ehsantn) [inline quickstart_notebook.ipynb:28]: Is this note really necessary? Isn't the cluster automatically attached by default?

File: quickstart_notebook.ipynb [hunk 2]
```diff
@@ -0,0 +1,1760 @@
...
+    "warnings.filterwarnings(\"ignore\")\n",
+    "\n",
+    "@bodo.jit\n",
+    "def read_data_nyc_taxi():\n",
+    "    df = pd.read_parquet(\n",
+    "        \"s3://bodo-example-data/nyc-taxi/yellow_tripdata_2019_half.pq\",\n",
+    "        )\n",
+    "    return df\n",
+    "\n",
+    "df = read_data_nyc_taxi()"
```
Comment (ehsantn) [inline quickstart_notebook.ipynb:108]: `TablePath` API allows passing a Parquet file directly, so this cell is not necessary.

File: quickstart_notebook.ipynb (hunk 1 above)
Comment (ritwika314) [reply quickstart_notebook.ipynb:28]: If people aren't immediately using their hosted trial workspace after signing up, the cluster is paused on the next sign in and detached. On unpausing, the cluster automatically attaches.

File: quickstart_notebook.ipynb [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+    "</br>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Extra: Pandas Feature Engineering  \n",
```
Comment (ehsantn) [inline quickstart_notebook.ipynb:159]:
```suggestion
    "### Pandas Feature Engineering  \n",
```

File: quickstart_notebook.ipynb [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info alert\" Note:\n",
+    "     \n",
+    "### note\n",
+    "\n",
+    "- The `%%px` cell [magic](\"https://ipyparallel.readthedocs.io/en/latest/tutorial/magics.html\") indicates that the code cell should run on all the cores of the cluster.\n",
+    "- The `@bodo.jit` decorator is an annotation to tell the bodo engine to parallelize the code. \n",
```
Comment (ehsantn) [inline quickstart_notebook.ipynb:224]:
```suggestion
    "- The `@bodo.jit` decorator is an annotation to tell the bodo engine to parallelize and optimize the code. \n",
```

**Review by ehsantn — APPROVED** [review]
Looks good to me.

**Pull request merged**